File: src/main/java/com/devspace/conexfy/ConexfyApplication.java
package com.devspace.conexfy;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

import com.ulisesbocchio.jasyptspringboot.annotation.EnableEncryptableProperties;

@SpringBootApplication
@EnableEncryptableProperties
public class ConexfyApplication {

	public static void main(String[] args) {
		SpringApplication.run(ConexfyApplication.class, args);
	}

}

File: src/main/kotlin/com/devspace/conexfy/ConexfyApplication.kt
package com.devspace.conexfy

import com.ulisesbocchio.jasyptspringboot.annotation.EnableEncryptableProperties
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableEncryptableProperties
class ConexfyApplication

fun main(args: Array<String>) {
    runApplication<ConexfyApplication>(*args)
}